src/gc/gcinterface.h

```diff
@@ -41,6 +41,51 @@ typedef enum
     walk_for_loh = 3
 } walk_surv_type;
 
+// Arguments to GCToEEInterface::StompWriteBarrierResize and
+// GCToEEInterface::StompWriteBarrierInitialize. 
+struct WriteBarrierResizeArgs 
```

Comment:
I think there should be only one struct that contains all write barrier parameters; the different types of callbacks can fill only some of them.

Otherwise, this will suffer from combinatoric growth as we add more different write barrier parameters (e.g. for the software write watch), and potentially more in future.